src/index.html

```diff
@@ -1,13 +1,233 @@
+        <div class="now-on-view__gallery--title">
+          <p class="gallery__title--title">Bodhisattvas of Wisdom, Compassion, and Power</p>
+          <p class="gallery__title--date">March 27, 2021 – October 30, 2022</p>
+        </div>
+      </div>
+      <div class="now-on-view__container">
+        <div class="now-on-view__photos">
+          <a href="#" class="now-on-view__link now-on-view__link--square">
+            <img class="now-on-view__photo" src="images/image6-square.png" alt="image">
+          </a>
+          <a href="#" class="now-on-view__link now-on-view__link--wide">
+            <img class="now-on-view__photo" src="images/image7-wide.png" alt="image">
+          </a>
+        </div>
+        <div class="now-on-view__gallery--title">
+          <p class="gallery__title--title">Inspiring Walt Disney: The Animation of French Decorative Arts</p>
+          <p class="gallery__title--date">December 10, 2021 – March 6, 2022</p>
+        </div>
+      </div>
+      <button class="button" href="#">View all exhibitions</button>
+    </section>
+
+    <section class="explore-online" id="explore-online">
+      <h2>More to explore online</h2>
+      <div class="explore-online__photos">
+        <a href="#" class="explore-online__link">
+          <img class="explore-online__photo" src="images/image8-square.png" alt="image">
+        </a>
+        <div class="explore-online__text">
+          <h3>Collection Areas</h3>
+          <p>These collection areas study, exhibit, and care for the objects in The Met collection.</p>
+        </div>
+      </div>
+      <div class="explore-online__photos">
+        <a href="#" class="explore-online__link">
+          <img class="explore-online__photo" src="images/image9-square.png" alt="image">
+        </a>
+        <div class="explore-online__text">
+          <h3>Travel</h3>
+          <p>Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.</p>
+        </div>
+      </div>
+      <div class="explore-online__photos">
+        <a href="#" class="explore-online__link">
+          <img class="explore-online__photo" src="./images/image10-square.png" alt="image">
+        </a>
+        <div class="explore-online__text">
+          <h3>Virtual group tours</h3>
+          <p> Request a guided tour, reserve admission and lead your own small group, or request a live virtual tour led by a Museum guide.</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="contacts" id="contacts">
+      <h2>Contact us</h2>
+      <div class="contacts__container__wrapper">
+        <div class="contacts__container">
+          <div class="contacts__container--card">
+            <p class="contacts__title">The Met Fifth Avenue</p>
+            <div class="contacts__wrapper">
+              <p class="contacts__title--title">Phone</p>
+              <a href="tel:+1 212-535--7710" class="contacts__title--content">+1 212-535--7710</a>
```

Comment:
The text color of phone email and address should change on hover [Checklist #4](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Change%20text%20color%20on%20hover%20for%20phone%2C%20email%2C%20and%20address)